views/js/qtiCreator/helper/xincludeRenderer.js

```diff
@@ -64,8 +72,7 @@ define([
                         //reload the wiget to rfresh the rendering with the new href
                         xincludeWidget.refresh();
                     }, loadedClasses);
-
-                    _.each(xincludeHandlers, handler => handler(xinclude.attr('href')));
+                    _.each(xincludeHandlers, handler => handler(xinclude.attr('href'), className, xi.serial));
```

Comment:
**suggestion**: we can replace underscore code with native code as well in the future conversations to ES6